controllers/vsphereclusteridentity_controller.go

```diff
@@ -73,8 +73,6 @@ type clusterIdentityReconciler struct {
 }
 
```

Comment:
We should also replace ResourceNotPausedAndHasFilterLabel here, right?

Reply:
It is using EnsurePausedCondition, how does it not set the condition?

Reply:
Ahh, mixed up with the other controllers 🤦